gluten-core/src/main/scala/io/glutenproject/execution/BatchScanExecTransformer.scala

```diff
@@ -48,6 +50,8 @@ class BatchScanExecTransformer(
   override def filterExprs(): Seq[Expression] = scan match {
     case fileScan: FileScan =>
       fileScan.dataFilters ++ pushdownFilters
+    case scan if scan.getClass.getSimpleName == "SparkBatchQueryScan" =>
```

Comment:
Why not use isInstanceOf ?

Reply:
SparkBatchQueryScan is protected class in iceberg, we can not import this class here.